### PR TITLE
Add DaemonSet -> Pods relation

### PR DESCRIPTION
**Pull request by roytman**

### CODE REVIEW

File: etc/skydive.yml.default [hunk 1]
```diff
@@ -113,7 +113,7 @@ analyzer:
 
     # list of probes used by the analyzers
     probes:
-      # - k8s
+      - k8s
```
Comment (hunchback) [inline etc/skydive.yml.default]: revert

File: etc/skydive.yml.default [hunk 2]
```diff
@@ -211,6 +212,7 @@ agent:
       # - libvirt
       # - runc
       # - vpp
+      - k8s
```
Comment (hunchback) [inline etc/skydive.yml.default]: revert

File: etc/skydive.yml.default [hunk 3]
```diff
@@ -529,13 +531,14 @@ ui:
     # Pre-defined Gremlin expression used in the WebUI for Filtering and Highlighting.
     # Note: Key should be in lower case
     favorites:
-    #   namespaces: "g.V().Has('Type', 'netns').OutE().BothV()"
-    #   layer2: "g.E().Has('RelationType', 'layer2')"
+      namespaces: "g.V().Has('Type', 'netns').OutE().BothV()"
+      layer2: "g.E().Has('RelationType', 'layer2')"
+      names: "g.V().Has('Type', 'Manager')"
```
Comment (hunchback) [inline etc/skydive.yml.default]: revert all the above

File: etc/skydive.yml.default [hunk 4]
```diff
@@ -529,13 +531,14 @@ ui:
...
 
     # Highlight Gremlin expression used by default and applied on WebUI load.
     # default_highlight: "layer2"
     # Filter Gremlin expression used by default and applied on WebUI load.
-    # default_filter: "layer2"
+    default_filter: "names"
```
Comment (hunchback) [inline etc/skydive.yml.default]: revert

File: etc/skydive.yml.default [hunk 5]
```diff
@@ -554,7 +557,7 @@ ui:
   # ssh_enabled: false
 
   # Enable/disable k8s related elements
-  # k8s_enabled: false
+  k8s_enabled: true
```
Comment (hunchback) [inline etc/skydive.yml.default]: revert

File: topology/probes/k8s/daemonset.go [hunk 6]
```diff
@@ -48,3 +50,13 @@ func (h *daemonSetHandler) Map(obj interface{}) (graph.Identifier, graph.Metadat
...
+
+func daemonSetPodAreLinked(a, b interface{}) bool {
+	ds := a.(*v1beta1.DaemonSet)
+	pod := b.(*v1.Pod)
+	return MatchNamespace(pod, ds) && matchLabelSelector(pod, ds.Spec.Selector)
+}
+
+func newDaemonSetPodLinker(g *graph.Graph) probe.Handler {
+	return NewABLinker(g, Manager, "daemonset", Manager, "pod", daemonSetPodAreLinked)
+}
```
Comment (hunchback) [inline topology/probes/k8s/daemonset.go]: did you run via gofmt ?

**Comment by roytman** [comment]
All required changes are done

**Comment by hunchback** [comment]
run tests

**Comment by hunchback** [comment]
run tests

**Comment by lebauce** [comment]
run tests

**Comment by lebauce** [comment]
run functional-tests-backend-orientdb
run scale-tests

**Pull request merged**